src/mlpack/methods/preprocess/preprocess_binarize_main.cpp

```diff
@@ -2,7 +2,7 @@
  * @file preprocess_binarize_main.cpp
  * @author Keon Kim
  *
- * binarize CLI executable
+ * binarize CMD executable
```

Comment:
```suggestion
 * A binding to binarize a dataset.
```
Here's another one we can improve too. :)